.packit.yaml

```diff
@@ -22,6 +22,15 @@ jobs:
     - epel-9
     enable_net: False
 
+- job: tests
+  trigger: pull_request
+  metadata:
```

Comment:
We've got rid of this `metadata` key -- you can still use it but you also don't need to. (Since more or less all existing configs out there has this key, we will not remove support shortly.)

So you can set all the subkeys directly on `job` level.